terraform/testcases/prometheus_static_adot_operator_awsprw/otconfig.tpl

```diff
@@ -0,0 +1,24 @@
+receivers:
+      prometheus:
+        config:
```

Comment:
Is this tab spacing correct?

Reply:
See this [comment](https://github.com/aws-observability/aws-otel-test-framework/pull/606#discussion_r840787649) above